recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
 
 test:
   commands:
-    - ${PREFIX}/bin/projects/electronMicroscopy/matchSeries --help
+    - matchSeries --help
```

Comment:
matchSeries does not support "--help". If you run it like this, it will report that it can't open the file "--help".

Reply:
I guess it is fine, because it reports: 
```
ParameterParser: Can't open file "--help". : file /home/conda/feedstock_root/build_artifacts/match-series_1592862563103/work/quocmesh/modules/aol/parameterParser.cpp, line 71
```
Which at least confirms that there is no issue with linking the libraries - that is the main thing I care for at the moment.

Reply:
I see, for that purpose it should be fine.

Reply:
I updated the test now by using the files you send me https://github.com/conda-forge/match-series-feedstock/pull/4